betterrolls5e/scripts/init.js

```diff
@@ -224,9 +224,7 @@ Hooks.on("getChatLogEntryContext", (html, options) => {
 		let option = options[i];
 		console.log(option);
 		if (contextDamageLabels.includes(option.name)) {
-			option.condition = li => canvas.tokens.controlledTokens.length && li.find(".dice-roll").length && !li.find(".red-full").length;
+			option.condition = li => canvas.tokens.controlled.length && li.find(".dice-roll").length && !li.find(".red-full").length;

```

Comment:
Let me know how backward compatible you'd like to remain, maybe we can switch the getter here based on foundry being < `0.6.0` or something.

Reply:
No worries on backward compatibility - New updates to the module will be for Foundry 0.6+. Thanks for the fix!